crypto/ed25519_test.go

```diff
@@ -3,6 +3,8 @@ package crypto
 import (
 	"reflect"
 	"testing"
+
+	ed25519 "golang.org/x/crypto/ed25519"
```

Comment:
This is now `crypto/ed25519`. See second paragraph: https://godoc.org/golang.org/x/crypto/ed25519

https://godoc.org/crypto/ed25519

Reply:
true, I will change it